src/components/base/BaseCard.vue

```diff
@@ -9,14 +9,11 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Prop, Vue } from 'vue-property-decorator';
 
-@Component({
```

Comment:
You still need the `@Component` decorator, just that `name` is not necessary.

Also, I recommend to show at least one use of `BaseCard` along with its definition.

Reply:
LOL, I want to keep the name field,

How should I present a BaseCard example, I will use base-card for refactoring in my later submissions, but due to the length of the PR review, I only commit this component for now(Is that enough, or do I need to create a separate example file?)